code/src/App.js

```diff
@@ -1,12 +1,33 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from 'Album'
+import { artists } from './artists'
+import { Header } from './Header'
+
+
+
+/*I console.log(data)*/
+/*console.log(Album)*/
 
-console.log(data)
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+      <div>
+        <Header />
+
+      </div>
+      <section className="album-container">
+        {data.albums.items.map(item => {
+          return <Album
+            key={item.id}
+            image={item.images[1].url}
+            name={item.name} artists={item.artists[0].name}
+            link={item.external_urls.spotify}
+            artists_link={item.artists[0].external_urls.spotify} />
```

Comment:
Props should be written in camelcase, so this last one should be `artistsLink={item.artists[0].external_urls.spotify}`